### ISSUE TITLE
DemoExampleMaker: documentation

### ISSUE THREAD

**Issue by nbriz** [netizenorg/netnet.studio#238]
once we create a new place for the `/docs` && start migrating the stuff from the Wiki over there, we're gonna drastically [need to update these docs](https://github.com/netizenorg/netnet.studio/wiki/code-examples) to reflect all the new changes @Cthomp7 has worked on

**Comment by Cthomp7** [comment]
@nbriz Do you still need someone to update this documentation? I happy to do so if needed :)

**Comment by nbriz** [comment]
@Cthomp7 yes! most definitely, we've shared the tool with a few folks, but having some updated docs for this will make it way more accessible!
